### PR TITLE
Add pacman simplistic tests

### PR DESCRIPTION
**Pull request by hassony2**
Simplistic tests to get acquainted with pytest fixtures

### CODE REVIEW

File: test/.cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "test_game.py::test_answer": true
+}
```
Comment (louismartin) [inline test/.cache/v/cache/lastfailed:3]: Careful with `git add .` :)

File: test/test_pacman.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+def test_pacman_initialized_at_zero(pacman):
+    assert pacman.reward == 0
+
+
+def test_pacman_move(node, pacman):
+    pacman.move(node)
+    assert(pacman.current_node.position == node.position)
+
+
+@pytest.fixture
```
Comment (louismartin) [inline test/test_pacman.py:14]: I think it comes from your text editor

File: test/test_pacman.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+def pacman():
+    from pacman.board import Node
+    from pacman.agents import PacMan
```
Comment (louismartin) [inline test/test_pacman.py:17]: Isn't weird to put the imports inside the fixture ?

**Review by louismartin — APPROVED** [review]

File: .gitignore [hunk 4]
```diff
@@ -1 +1,2 @@
 __pycache__
+test/.cache
```
Comment (louismartin) [inline .gitignore:2]: Did you remove the folder from git too ?
You could have put just `.cache` this way it ignores `.cache` in all folders.

File: test/.cache/v/cache/lastfailed (hunk 1 above)
Comment (hassony2) [reply test/.cache/v/cache/lastfailed:3]: I did git add test/
but still... !

Comment (louismartin) [reply test/.cache/v/cache/lastfailed:3]: Haha you either need to add specific files (which I do) or check with `git status` that you added the right files (which you can always do to double check).
If you want to check the diff of your staged files you can always do `git diff --cached`

**Review by louismartin — APPROVED** [review]

**Pull request closed without merging**